crates/sui-client-gen/src/framework_sources.rs

```diff
@@ -0,0 +1,843 @@
+pub static ESLINTRC: &str = r#"{
+    "rules": {
+        "@typescript-eslint/ban-types": "off",
+        "@typescript-eslint/no-explicit-any": "off",
+        "@typescript-eslint/no-unused-vars": "off"
+    }
+}
+"#;
+
+pub static LOADER: &str = r#"
+import { compressSuiType, parseTypeName } from './util'
```

Comment:
it might be cleaner to publish all the static/shared files as a new npm package rather than generating them out.    The big downside with this is versioning, since now you have a dependency between the generator and the package that has the shared definitions.   It wouldn't be much different than the dependency on @mysten/sui.js that already exists, so that might be fine